package.json

```diff
@@ -38,7 +38,7 @@
     "lie": "3.0.4",
     "pouchdb-extend": "0.1.2",
     "pouchdb-promise": "5.4.3",
-    "pouchdb-wrappers": "4.1.0",
+    "pouchdb-wrappers": "https://github.com/neighbourhoodie/pouchdb-wrappers.git#mergemaster",
```

Comment:
This is a fix for https://github.com/pouchdb-community/transform-pouch/issues/55. Maybe there will be an update soon